R/read_data.R

```diff
@@ -26,12 +26,6 @@ data_choices <- function(data, column) {
 # Note that all of these that read a parquet file do a 'lazy read'
 # you will need to use `%>% collect()` to pull the final table into memory
 
-## Provider breakdowns --------------------------------------------------------
```

Comment:
These lines need adding back in otherwise the first page of the dashboard can't read any data - think they went missing in the conflict resolution